src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -28,9 +29,16 @@ public void SslStream_StreamToStream_Authentication_Success()
                 Task[] auth = new Task[2];
                 auth[0] = client.AuthenticateAsClientAsync(certificate.Subject);
                 auth[1] = server.AuthenticateAsServerAsync(certificate);
-
+                
                 bool finished = Task.WaitAll(auth, TimeSpan.FromSeconds(3));
                 Assert.True(finished, "Handshake completed in the allotted time");
+
+                string testData = "Hello SSL";
```

Comment:
This test seemed to be just doing a handshake.  Doing a read/write seem like a separate (additional) test.